### ISSUE TITLE
Make init fails

### ISSUE THREAD

**Issue by geoffroya** [elastic/beat-generator#32]
Hi all

I want to create a simple beat. I followed the README:

```
cookiecutter https://github.com/elastic/beat-generator.git
```

I installed GO 1.6 and set $GOPATH:

```
$ go version
go version go1.6 linux/amd64
```

When first trying to run ```make init``` I figured out that glide was missing. So I unzipped ot and updated PATH.

Now when I do ```make init``` I have the following output?

```
$ make init
glide update  --no-recursive
[WARN] stat /home/garnoud/DEV/beats/src: no such file or directory
[ERROR] Could not find any of /home/garnoud/DEV/beats/src.
[INFO] As of Glide 0.5/Go 1.5, this is required.
[ERROR] Wihtout src, cannot continue.
Makefile:14: recipe for target 'init' failed
make: *** [init] Error 1
```

Thus I have some questions:
* my beat is under /home/garnoud/DEV/beats/stdin-beat (GOPATH=/home/garnoud/dev/beats), is that right?
* there is not **src** dir in the result of cookiecutter, is that right?

Thanks for your feedbacks

Geoff.

**Comment by gingerwizard** [comment]
I encountered this issue.  It should be clear that the src directory is required e.g. 
suppose my GOPATH=/opt/go-projects/

my project is github.com/gingerwizard/mybeat

I have the directory /opt/go-projects/src/github.com/gingerwizard/mybeat

Ideally we would create this directory @ruflin

**Comment by geoffroya** [comment]
Thank you!

**Comment by gingerwizard** [comment]
Leaving this open as we may want to action something here - probably documentation.

**Comment by geoffroya** [comment]
More than doc, I think that the src/github.com/id/project should be created automatically, shouldn't it?

**Comment by ruflin** [comment]
@geoffroya @gingerwizard I'm not 100% sure about the auto generation of the directory. The reason here that this is not something beat specific but Golang specific. I think best would be to improve the docs to first describe more in details the directory structure that must be used and link to the specific Golang docs. WDYT?

**Comment by gingerwizard** [comment]
@ruflin I agree. It would be nice if the generator also produced the full folder hierarchy i.e. github.com/<username>/<beatname>
Or do you consider this to be Golang specific?

**Comment by ruflin** [comment]
@gingerwizard I would call it Golang specific. But at the same time it could help quite a bit as most people that start a beat never have done any Golang before.
